ocs_ci/ocs/constants.py

```diff
@@ -1104,7 +1104,12 @@
     ROSA_PLATFORM,
     FUSIONAAS_PLATFORM,
 ]
-BAREMETAL_PLATFORMS = [BAREMETAL_PLATFORM, BAREMETALPSI_PLATFORM]
+BAREMETAL_PLATFORMS = [
+    BAREMETAL_PLATFORM,
+    BAREMETALPSI_PLATFORM,
+    HCI_BAREMETAL,
+    IBM_POWER_PLATFORM,
```

Comment:
@suchita-g , @nehaberry   I wanted to inform you about these modifications. Please let me know if they are causing any trouble with your existing tests.

Reply:
I could find only one usage which is https://github.com/paraggit/ocs-ci/blob/abb6e6792c0e6516baabd64726d58ddadc6a7c11/ocs_ci/deployment/factory.py#L71
The deployment automation on the platform which is mentioned as HCI_BAREMETAL is not completed. May be the the use of HCI_BAREMETAL is not causing a problem but I doubt if that is required in the list seeing the usage scenarios.